zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -176,24 +232,28 @@ object Cookie {
     val cookies: Array[String]  = headerValue.split(";").map(_.trim)
     val x: List[Option[Cookie]] = cookies.toList.map(a => {
       val (name, content) = splitNameContent(a)
-      if (name.trim == "" && content.isEmpty) None
-      else Some(Cookie(name, content.getOrElse("")))
+      if (name == "" && content.isEmpty) None
+      else Some(Cookie(name, content))
     })
 
     if (x.contains(None))
       None
     else Some(x.map(_.get))
   }
 
-  private def parseDate(v: String): Option[Instant] =
-    Try(Instant.parse(v)).toOption
-
-  private def splitNameContent(kv: String): (String, Option[String]) =
-    kv.split("=", 2).map(_.trim) match {
-      case Array(v1)     => (v1, None)
-      case Array(v1, v2) => (v1, Some(v2))
-      case _             => ("", None)
+  @inline
+  private def parseDate(v: String): Try[Instant] =
+    Try(Instant.parse(v))
```

Comment:
Try will allocate additional memory. Instead, you could use a `try { } catch {}` block.
Incase of failure return a `null`